### ISSUE TITLE
Running `composer install` doesn't produce md2resume_dev.php file

### ISSUE THREAD

**Issue by ejwaibel** [there4/markdown-resume#71]
```
$ composer install
Loading composer repositories with package information
Installing dependencies (including require-dev) from lock file
Nothing to install or update
Generating autoload files
```

After command is finished, the `md2resume_dev.php` file doesn't get created.

**Comment by kaaljabr** [comment]
I found the issue it is line number 13 of md2resume file

```
$autoloadPath = basename($baseDir) === 'markdown-resume'
              ? realpath($baseDir . '/../../autoload.php')
              : $baseDir . '/vendor/autoload.php';
```
executing this piece of code sets null or undefined in $autoloadPath.

change it to 
```
$autoloadPath = $baseDir . '/vendor/autoload.php';
```
it should work

**Comment by spawnia** [comment]
Should by resolved by #79
